src/Workspaces/Core/Portable/CodeFixes/CodeFixContext.cs

```diff
@@ -48,7 +48,13 @@ public struct CodeFixContext
         /// </summary>
         /// <param name="document">Document to fix.</param>
         /// <param name="span">Text span within the <paramref name="document"/> to fix.</param>
-        /// <param name="diagnostics">Diagnostics to fix. All the diagnostics should have the same span <paramref name="span"/>.</param>
+        /// <param name="diagnostics">
+        /// Diagnostics to fix. All the diagnostics should have the same span <paramref name="span"/>.
+        /// <para>
+        /// All the provided diagnostics must have IDs that are fixable by the associated <see cref="CodeFixProvider"/>, as indicated by its <see cref="CodeFixProvider.FixableDiagnosticIds"/> property.
+        /// Otherwise, the behavior of <see cref="CodeFixProvider.RegisterCodeFixesAsync(CodeFixContext)"/> is undefined when invoked with this context object.
```

Comment:
The statement beginning with "Otherwise" can be omitted without loss of meaning.